hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/BaseUIEntityEvent.java

```diff
@@ -31,6 +46,29 @@
     public BaseUIEntityEvent(final BaseEntityEventType eventType, final T entity) {
         this.eventType = eventType;
         this.entity = entity;
+        entityIds = new ArrayList<>();
+        if (entity != null) {
+            entityIds.add(entity.getId());
+            this.entityClass = entity.getClass();
+        }
+    }
+
+    /**
+     * Delete entity event.
+     * 
+     * @param entityIds
+     *            entities which will be deleted
+     * @param entityClass
+     *            the entityClass
+     */
+    public BaseUIEntityEvent(final Collection<Long> entityIds, final Class<T> entityClass) {
+        this.eventType = BaseEntityEventType.REMOVE_ENTITIES;
```

Comment:
By the way: `SoftwareModuleEvent(BaseEntityEventType.REMOVE_ENTITIES, new SoftwareModule())` do the same. It will add a a single id to the list instead of a collection.